src/features/public-api/components/CreateApiKeyButton.tsx

```diff
@@ -95,6 +103,29 @@ export function CreateApiKeyButton(props: { projectId: string }) {
           <div className="text-md mb-2 font-semibold">Public Key</div>
           <CodeView content={generatedKeys?.publicKey ?? "Loading ..."} />
         </div>
+        {generatedKeys?.publicKey && generatedKeys.secretKey && (
+          <div className="mb-2">
+            <div className="text-md my-2 font-semibold">Usage</div>
+            <Tabs defaultValue="javaScript">
+              <TabsList>
+                <TabsTrigger value="javaScript">JS/TS</TabsTrigger>
+                <TabsTrigger value="python">Python</TabsTrigger>
+              </TabsList>
+              <TabsContent value="javaScript">
```

Comment:
Just thought about it. What about adding a lonk to the docs here based on the tab?